docs/amber_script.md

```diff
@@ -213,6 +213,14 @@ The following commands are all specified within the `PIPELINE` command.
   END
 ```
 
+```groovy
+  # Set the compile options used to compile the given shader. Currently, only
+  # support for OPENCL-C shaders.
+  COMPILE_OPTIONS {shader_name}
+    {option}+
+  END
+```
+
```

Comment:
Can you add an example here of what the compile options for clspv would look like? How do we handle options with arguments?

Reply:
I add an OpenCL example in the examples section. It includes an option with an argument. Also added a note that arguments are parsed the same as on the command line.